python/test/unit/test_profiling.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file won't be picked up until it is added to the 'runner.py' script here: https://github.com/arbor-sim/arbor/blob/master/python/test/unit/runner.py